CalculatorWithTests/Calculator.cs

```diff
@@ -1,8 +1,43 @@
-﻿
+﻿using System;
+using System.Linq;
+
 namespace CalculatorWithTests
 {
-    class Calculator
+    public class Calculator
     {
-        
+        public decimal GetSum(decimal number1, decimal number2)
+        {
+            return number1 + number2;
+        }
+
+        public decimal GetDifference(decimal number1, decimal number2)
```

Comment:
For subtraction, the order of the numbers matters. These parameters could have more descriptive names to indicate which number is being subtracted from which.